web/pages/characters/[name].tsx

```diff
@@ -57,6 +57,12 @@ const CharacterInfo = ({ data, name }: Props) => {
               </div>
             </HStack>
 
+            {/* If null, don't do anything. Refer to 'db/characters.tsx' in the Weekly Materials. */}
+            {/* The 'null' value is possible on unreleased characters. We have no idea about their weeklies. */}
+            {character.talentMaterialWeekly?.map((material) => (
```

Comment:
Same comment as above. `map` would return an empty array which doesn't render anything anyway so I don't see the benefit of null.

Reply:
This too! Sorry! Resolved.